### PR TITLE
Integrate front and back end

### PR DESCRIPTION
**Pull request by Lijia-Li**
### Changes been made
- change front template by inserting form
- change back end template to display search result
- add Flask and view function to combine front and back end

### Sample result page: 
screen shot 2018-03-25 at 10 23 19 am

@justinnhli @crzeller11

### CODE REVIEW

File: ReCord_frontend.py [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
 def my_form():
-    return render_template('search_test_2.html')
+    return render_template('ReChord_front.html')
+
 
 @app.route('/', methods=['POST'])
 def my_form_post():
-    text = request.form['text']
-    return transform_xml(text)
+    if request.form['submit'] == 'Search Snippet':
```
Comment (justinnhli) [inline ReCord_frontend.py]: This function needs a docstring.

File: ReCord_frontend.py [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
-def transform_xml(xml_code):
-    xml = xml_code.encode()
-    f = open('upload.xml', 'wb')
-    f.write(xml)
-    f.close()
 
 @app.route('/')
 def my_form():
-    return render_template('search_test_2.html')
+    return render_template('ReChord_front.html')
```
Comment (justinnhli) [inline ReCord_frontend.py]: This function needs a docstring

**Review by justinnhli — COMMENTED** [review]
Just some quick changes and we're done.

**Comment by Lijia-Li** [comment]
@justinnhli The docstrings have been added.

**Pull request merged**